lib/network_utils.ml

```diff
@@ -60,7 +60,22 @@ let igmp_query_maxresp_time = ref "5000"
 let enable_ipv6_mcast_snooping = ref false
 let mcast_snooping_disable_flood_unregistered = ref true
 
-let check_n_run run_func script args =
+let default_error_handler script args stdout stderr exn =
+  let message =
+    match exn with
+    | Unix.WEXITED n -> Printf.sprintf "Exit code %d" n
+    | Unix.WSIGNALED s -> Printf.sprintf "Signaled %d" s (* Note that this is the internal ocaml signal number, see Sys module *)
+    | Unix.WSTOPPED s -> Printf.sprintf "Stopped %d" s
```

Comment:
Should this include a wildcard for other exceptions?

Reply:
Hmm... this is not actually an exception, but a value of type `Unix.process_status`, which only has these three constructors. I'll clarify the names.